### PR TITLE
Impact Crater logic

### PR DESCRIPTION
**Pull request by gollop**
Doesn't update the human-readable version. The tricks component of the JSON is just echoes tricks with some echoes-exclusive tricks removed.

### CODE REVIEW

File: randovania/data/json_data/prime1.json [hunk 1]
```diff
@@ -788,7 +1433,122 @@
                             "connected_dock_index": 1,
                             "dock_type": 0,
                             "dock_weakness_index": 0,
-                            "connections": {}
+                            "connections": {
+                                "Dock to Subchamber Four": {
+                                    "type": "and",
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime1.json:1438]: This is being a big and: `Power Beam and Ice Beam and Wave Beam and Plasma Beam and Missile ≥ 50 and Flamethrower and Charge Beam and Knowledge (Beginner)`

File: randovania/data/json_data/prime1.json [hunk 2]
```diff
@@ -803,33 +1563,148 @@
...
                             "connected_area_asset_id": 2050677022,
                             "connected_dock_index": 0,
-                            "dock_type": 0,
-                            "dock_weakness_index": 0,
-                            "connections": {}
+                            "dock_type": 2,
+                            "dock_weakness_index": 0,
+                            "connections": {
+                                "Dock to Subchamber Five": {
+                                    "type": "and",
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime1.json:1594]: this is being just a big "and": `Power Beam and Ice Beam and Wave Beam and Plasma Beam and Missile ≥ 50 and Flamethrower and Charge Beam and Knowledge (Beginner)`

File: randovania/data/json_data/prime1.json [hunk 3]
```diff
@@ -448,18 +563,18 @@
...
+                                "y": -29.9,
+                                "z": 2.5
                             },
                             "node_type": "dock",
                             "dock_index": 0,
                             "connected_area_asset_id": 1238049635,
                             "connected_dock_index": 1,
                             "dock_type": 0,
-                            "dock_weakness_index": 0,
+                            "dock_weakness_index": 1,
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime1.json:577]: The dock_weakness_database has the same index for Ice/Wave/Plasma doors, so this is likely wrong :X

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

**Comment by gollop** [comment]
UltiNaruto implemented no tricks logic for this, so closing.

**Pull request closed without merging**